compose.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
This script feels a bit unnecessary to me. Anyone deploying a container using docker-compose should already know how to do so. I'll let you and andris be the judge of that though.

Reply:
Force of habit, less typing, will remove